tests/parametric/test_128_bit_traceids.py

```diff
@@ -122,14 +121,13 @@ def test_datadog_128_bit_propagation_tid_chars(self, test_agent, test_library):
         assert dd_p_tid is None
 
     @missing_feature(context.library == "dotnet", reason="Optional feature not implemented")
-    @missing_feature(context.library == "golang", reason="Optional feature not implemented")
     @missing_feature(context.library == "nodejs", reason="not implemented")
     @missing_feature(context.library == "php", reason="Issue: Traces not available from test agent")
     @missing_feature(context.library == "python_http", reason="not implemented")
     @missing_feature(context.library == "ruby", reason="not implemented")
     @missing_feature(
         context.library == "golang",
-        "XXXX tag would be discarded by the length rule;"
+        reason="XXXX tag would be discarded by the length rule;"
```

Comment:
I would actually keep `reason="Optional feature not implemented")`, and remove the lines 128-132, since this test only checks the presence of the error, which we don't set.

Reply:
SGTM!

Reply:
PR updated!